src/components/AboutProject/useStyles.js

```diff
@@ -0,0 +1,55 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(({ breakpoints, typography, palette }) => ({
+  root: {
+    height: typography.pxToRem(600),
+    margin: 0,
+    position: "relative",
+    [breakpoints.up("md")]: {
+      height: typography.pxToRem(400),
+    },
+  },
+  background: {
+    position: "absolute",
+    top: 0,
+    bottom: 0,
+    height: typography.pxToRem(600),
+    width: "100%",
+    zIndex: -1,
+    [breakpoints.up("lg")]: {
+      height: typography.pxToRem(400),
+    },
+  },
+  section: {
+    zIndex: 1,
+    height: "100%",
+  },
+  button: {
+    textDecoration: "none",
+    "&:hover": {
+      textDecoration: "none",
+    },
```

Comment:
Aren't these on theme already? If not, that's where they should be. _Unless_ you're saying this behaviour is unique to this component.